actions/knowledge/filehelper.ts

```diff
@@ -28,6 +30,28 @@ export async function getFileOrFolderSizeInKB(
   return 0;
 }
 
-export async function getBasename(filePath: string): Promise<string> {
-  return path.basename(filePath);
+export async function importFiles(
+  files: string[]
+): Promise<Map<string, FileDetail>> {
+  const result: Map<string, FileDetail> = new Map();
+
+  for (const file of files) {
+    // check if filepath lives in notion or onedrive integration folders
+    const baseDir = path.dirname(path.dirname(file));
```

Comment:
Why are we calling path.dirname twice? What is the path we're retrieving? Whatever the answer, it could be a good comment.